docker/deploy-contracts/prepare_db.sh

```diff
@@ -33,21 +33,8 @@ cd -
 
 # deploy Wrapped1155Factory contract
 cd 1155-to-20
-echo "module.exports = {
-  networks: {
-    development: {
-      host: \"ganache\",
-      port: 8545,
-      network_id: \"*\",
-    }
-  },
-  compilers: {
-    solc: {
-      version: \"0.6.12\",
-    }
-  }
-}" > truffle-config.js
-./node_modules/.bin/truffle migrate --network development
+sed -i 's/localhost/ganache/g' truffle-config.js
+./node_modules/.bin/truffle migrate --network local
```

Comment:
It could be that you are editing `truffle-config.js` with sed but you're not creating it before?, in line 49 you were pipe to it but that line is removed now.

Reply:
The `truffle-config.js` file now comes from 1155-to-20 repo, the sed replaces local endpoint to ganache docker container. Anyway, I have to update the repository to latest version after one update, so I changed title to WIP. Thanks @hpmaxi !